Techome/Techome/Views/Trend/TrendView.swift

```diff
@@ -0,0 +1,221 @@
+//
+//  TrendView.swift
+//  Techome
+//
+//  Created by 한택환 on 2022/06/09.
+//
+
+import SwiftUI
+
+struct TrendLayout {
```

Comment:
```suggestion
struct TrendViewLayoutValue {
```

지금 명명이라면 Layout을 설정하는 method가 있는 건지, property가 있는것인지 구분이 되지 않아서요 ..!
위와 같이 구조체 이름이 좀 더 명확하면 좋을 것 같아요 ~

Reply:
확실히 구분이 안되는 것 같아요. 구조체 이름 좀 더 명확하게 수정하겠습니다!!